content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+| `:organization_name` | The name of the organization the provider belongs to.                                                    |
+| `:namespace`         | The namespace of the provider. Must be the same as the `organization_name` for the provider              |
+| `:name`              | The provider name.                                                                                       |
+| `:version`           | The version of the provider.                                                                             |
+| `:os`                | The os of the provider platform.                                                                         |
+| `:arch`              | The architecture of the provider platform.                                                               |
+| `:registry_name`     | Must be `private`. |
+
+| Status  | Response                                             | Reason                                                    |
+| ------- | ---------------------------------------------------- | --------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "registry-providers"`) | The request was successful                                |
+| [403][] | [JSON API error object][]                            | Forbidden - public provider curation disabled             |
+| [404][] | [JSON API error object][]                            | Provider not found or user unauthorized to perform action |
+
+### Sample Request (private provider)
+
+```shell
+curl \
+  --request GET \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms/linux/amd64
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "provpltfrm-BLJWvWyJ2QMs525k",
+    "type": "registry-provider-platforms",
+    "attributes": {
+      "os": "linux",
+      "arch": "amd64",
+      "filename": "terraform-provider-aws_3.1.1_linux_amd64.zip",
+      "shasum": "8f69533bc8afc227b40d15116358f91505bb638ce5919712fbb38a2dec1bba38",
+      "permissions": {
+        "can-delete": true,
+        "can-upload-asset": true
+      },
+      "provider-binary-uploaded": true
+    },
+    "relationships": {
+      "registry-provider-version": {
+        "data": {
+          "id": "provver-y5KZUsSBRLV9zCtL",
+          "type": "registry-provider-versions"
+        }
+      }
+    },
+    "links": {
+      "provider-binary-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+    }
+  }
+}
+```
+
+### Note
+
+`provider-binary-uploaded` may be `false` if that file hasn't been uploaded to Archivist yet. In this case, instead of including a link to `provider-binary-download`. The response will include an upload link `provider-binary-upload` instead.
```

Comment:
```suggestion
### Note

-> **Note:** The `provider-binary-uploaded` attribute will be `false` if that file hasn't been uploaded to Archivist yet. In this case, instead of including a link to `provider-binary-download`, the response will include an upload link `provider-binary-upload`.
```